### DISCUSSION TITLE
How to move the selection backward of it ?

### DISCUSSION THREAD

**Discussion by Dangerise** [helix-editor/helix#9453] [Q&A] [answered]
![selection](https://github.com/helix-editor/helix/assets/88434542/7b70bfc7-37c6-4400-b0bf-1ceecc56737b)

I can move the selection forward by tab button, but I don't know how to move backward.

**Comment by woojiq** [comment, marked as answer]
Shift-Tab?

**Comment by Dangerise** [comment]
Solved
